src/page_templates/errors/DefaultErrorPage.react.js

```diff
@@ -2,15 +2,15 @@
 
 import * as React from "react";
 
-import Container from "../components/Container.react";
-import Page from "../components/Page/Page.react";
-import Header from "../components/Header/Header.react";
-import Icon from "../components/Icon/Icon.react";
-import Button from "../components/Button/Button.react";
+import Container from "../../components/Container.react";
+import Page from "../../components/Page/Page.react";
+import Header from "../../components/Header/Header.react";
+import Icon from "../../components/Icon/Icon.react";
+import Button from "../../components/Button/Button.react";
 
-type Props = {|
-  +title: string,
-  +subtitle: string,
+export type Props = {|
+  +title?: string,

```

Comment:
I think it might be a good idea to leave these as required props, the error page would look kinda silly without them.  Just a thought though, I'm good with either way

Reply:
I agree but I've given them default values instead as it's so I could just reuse the type in the other error pages easily.